cni/network/plugin/main.go

```diff
@@ -18,21 +26,72 @@ var version string
 // Main is the entry point for CNI network plugin.
 func main() {
 	var config common.PluginConfig
+	var err error
 	config.Version = version
 
+	reportManager := &telemetry.ReportManager{HostNetAgentURL: hostNetAgentURL, IpamQueryURL: ipamQueryURL, ReportType: reportType}
+
```

Comment:
remove empty lines to follow existing code format  (applicable to rest of the review as well)